### ISSUE TITLE
GPIO question

### ISSUE THREAD

**Issue by Landsil** [3cky/flipperzero-airmon#2]
Hi,
Is it absolutely required to wire ground to pin 11 instead of pin 8?

I'm planning to wire small PI fan to 3v3 to force air into sensor and it would be neater to have it on pins 9 and 11 I think.
(yes I'm very new and this is my first project like that)

**Comment by 3cky** [comment]
> Is it absolutely required to wire ground to pin 11 instead of pin 8?

No, it's not required. Both 8 and 11 pins can be used as ground.
